tutorials/tmva/keras/ClassificationKeras.py

```diff
@@ -33,32 +32,36 @@
 dataloader.AddSignalTree(signal, 1.0)
 dataloader.AddBackgroundTree(background, 1.0)
 dataloader.PrepareTrainingAndTestTree(TCut(''),
-        'nTrain_Signal=4000:nTrain_Background=4000:SplitMode=Random:NormMode=NumEvents:!V')
+                                      'nTrain_Signal=4000:nTrain_Background=4000:SplitMode=Random:NormMode=NumEvents:!V')
 
 # Generate model
 
 # Define initialization
+
+
 def normal(shape, name=None):
```

Comment:
Remove this definition of the custom init, you've removed it from the model (that's fine!). As well, please clean up the spacings around.